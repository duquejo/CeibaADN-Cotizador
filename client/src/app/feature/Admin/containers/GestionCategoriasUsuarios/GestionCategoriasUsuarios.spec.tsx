import * as React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';
import { GestionCategoriasUsuarios } from './index';
import { SinonStub } from 'sinon';

describe('GestionCategoriasUsuarios Test', () => {

  let componentWrapper: ShallowWrapper;
  let componentProps: React.ComponentProps<typeof GestionCategoriasUsuarios> & {
    listarCategoriasUsuarios: SinonStub;
    agregarNuevaCategoriaUsuarios: SinonStub; 
  };

  it('DeberÃ­a renderizar el componente', () => {
    componentWrapper = shallow( <GestionCategoriasUsuarios { ...componentProps } /> );
    expect(componentWrapper).toMatchSnapshot();
  });

});