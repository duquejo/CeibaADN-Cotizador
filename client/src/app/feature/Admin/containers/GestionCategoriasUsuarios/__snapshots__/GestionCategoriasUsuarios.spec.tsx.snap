// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GestionCategoriasUsuarios Test Deber칤a renderizar el componente 1`] = `
<Fragment>
  <h2>
    Gesti칩n de categor칤as de usuarios 
    <span
      aria-label="Categor칤as de usuarios"
      role="img"
      title="Categor칤as de usuarios"
    >
      游녻
    </span>
  </h2>
  <p>
    Gestione las categor칤as de usuarios que podr치 asignar a los Centros Vacacionales, estas categor칤as tienen influencia sobre el valor de la cotizaci칩n que un usuario obtendr치.
  </p>
  <styled.div>
    <styled.div>
      <CrearCategoria />
    </styled.div>
    <styled.div>
      <AccionesCategorias />
    </styled.div>
  </styled.div>
  <styled.hr />
</Fragment>
`;
